src/AST/Declaration.cs

```diff
@@ -315,6 +315,8 @@ public virtual bool Ignore
         // another translation unit. 
         public string USR;
 
+        public List<Declaration> Redeclarations = new List<Declaration>();
```

Comment:
Do not use public fields. We do still have them in many places but we gradually replace them with properties. In this case it's really convenient to use C# 6.0 property initialisers:

    public List<Declaration> Redeclarations { get; } = new List<Declaration>();